src/main/java/core/basesyntax/service/CsvFormatter.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.TransactionList;
+import core.basesyntax.model.Transaction;
+
+public class CsvFormatter {
```

Comment:
```suggestion
public class TransactionParserImpl {
```

lets create some interface for this implementation (same for other services)